base58.py

```diff
@@ -27,9 +27,20 @@
     buffer = lambda s: s.buffer
 
 
+def int_to_b58(i):
+    '''Encode an integer using Base58'''
+    string = ""
+    while i:
+        idx = i % 58
```

Comment:
Because internally the other encoders convert to a number before encoding to base58 this should look exactly the same as the L57-L60 in b58encode